content/blog/2022-04-07-compiling-black-with-mypyc-part-3.md

```diff
@@ -0,0 +1,185 @@
+workflow run of the day, downloaded the artifacts, tested one of them locally to make sure
+nothing was on fire, and pushed 'em to PyPI and that's how release 22.1.0 was born 🎉.
+
+The core team chat was quite lively for the next hour to say the least, after all we just
+finished a major milestone! Also yeah, we *may* have done a lot of publicizing for this
+release which is why it was everywhere for a while :wink:
+
+### Post release calm
+
+To be honest I haven't really seen anyone comment on the fact Black is now compiled with
+mypyc yet short of this one [mini blog post][that-one-post-about-black-mypyc] ... which is
+both disappointing since I spent so much time on this effort, but also calming since
+usually buzz is sparked by bugs and whatnot.
+
+Having mentioned bugs and crashes, so far we've received two reports,
+[one bug probably from mypyc and an unintentional restriction on Black's unofficial APIs][not-too-many-fires-so-far].
+
+Version 22.1.0 still unintentionally broke quite a few integrations, though they
+weren't mypyc related. Turns out lots of people depend on `black.files.find_project_root`
+returning a `pathlib.Path` and nothing else! I've seen at least five issues / PRs on
+GitHub fixing crashes related to our change making it return a tuple instead.
+
+For this reason we, the core team, want to [define a stable API (GH-779)][gh-779] soon. I
+can't promise anything as Black's development is volunteer-based, but if you were curious
+to what's next for psf/black, there's this.
+
+## Results & final thoughts
+
+**Black is now overall 2 times faster**[^1]. And as a bonus, startup time is down too (at
+most by 15%). You can [read the whole report here][perf-report]. Please note these numbers
+were gathered quite a long time ago and **they are probably a bit outdated**, especially
+with the recent blib2to3 changes made to support 3.10 syntax.
+
+From my experience, I believe mypyc will make for an interesting, but viable solution to
+speeding up Python code going forward. It's far from perfect and still has a bunch of
+bugs, but today it is already making an impact. Should you use it in production if you
+don't want to feel like being a beta tester? Certainly not, but it's sure still possible
+and I'm happy with the results!
+
+I hope to see the mypyc project gain more contributors and succeed. Along with Cython,
+PyPy, and the recent faster-cpython project, **perhaps Python will have a solid speed
+story**.
+
+> And yes, I did indeed learn a ton embarking on this project. Made mistakes along the
+> way, but they turned out alright ❀
+
+Congrats on reaching the end of this multi-part blog post!
+
+## Acknowledgements
+
+I'd like to thank [@msullivan] for his original work on integrating mypyc into Black,
+spawning this summer project and eventually this blog series :)
+
+$try-getting-this-reviewed-by-mypyc-contributor
+
+$try-getting-this-reviewed-by-co-maintainers
```

Comment:
```suggestion
$try-getting-this-reviewed-by-co-maintainers
```
Not sure if you mean to have these in here, still.

Reply:
Yea... I forgot about this until after I pushed this. I'll replace this with proper credits soon.